pkg/webhooks/exception/validate.go

```diff
@@ -0,0 +1,27 @@
+package exception
+
+import (
+	"context"
+	"time"
+
+	"github.com/go-logr/logr"
+	admissionutils "github.com/kyverno/kyverno/pkg/utils/admission"
+	validation "github.com/kyverno/kyverno/pkg/validation/exception"
+	admissionv1 "k8s.io/api/admission/v1"
+)
+
+// TODO: wrap this into an interface passed at server creation time
```

Comment:
is this a valid TODO?

Reply:
Yes, we don’t want the server to directly call the implementation, it should go through an interface.